### PR TITLE
Update | Supports Module activity on Dashboard

### LINKED ISSUE #783: Bug: Empty Transaction Data in Homepage for child creation
### Overview

Not sure what's going on with this.. but the transaction was the creation of a child Usul DAO.

### Steps to reproduce

https://app.dev.fractalframework.xyz/#/daos/0xF332aAb56e00f7FDEdEDDC50033Ce5e2D0DD6d61

https://goerli.etherscan.io/tx/0x1ded9a9f739a85770fd792134e952ce17f88af823f95c67efc3778542f9348b0

### Screenshots / Video

Screenshot 2022-12-13 at 8 18 07 PM
 

### Version

Dev (app.dev.fractalframework.xyz/)

### Browser

Firefox

### Operating System

Mac

### web3 wallet

None

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description
Updates `useParsedTxs` hook to handle MODULE_TRANSACTION types.

## Issue / Notion doc (if applicable)
Closes #783 

## Screenshots (if applicable)
localhost_3000_(Desktop) (23)

### CODE REVIEW

File: src/components/ui/badges/Badge.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ const BADGE_MAPPING: BadgeType = {
   [TxProposalState.Canceled]: { Icon: CloseX, bg: 'sand.700', color: 'grayscale.black' },
   [TxProposalState.TimeLocked]: { Icon: ClockTwo, bg: 'sand.700', color: 'grayscale.black' },
   [TxProposalState.Uninitialized]: { Icon: CloseX, bg: 'sand.700', color: 'grayscale.black' },
+  [TxProposalState.Module]: { bg: 'sand.700', color: 'grayscale.black' },
```
Comment (Da-Colon) [inline src/components/ui/badges/Badge.tsx:20]: I went with no Icon, didn't see any in our current library that made sense

**Review by Da-Colon — COMMENTED** [review]
Adding the description into translation....done

File: src/providers/Fractal/governance/types.ts [hunk 2]
```diff
@@ -83,6 +83,7 @@ export interface GovernanceContracts {
 }
 
 export enum TxProposalState {
+  Module = 'stateModule',
```
[the remark this answers is not in the document]
Comment (Da-Colon) [reply src/providers/Fractal/governance/types.ts:86]: o thats right thanks for the reminder

[the remark this answers is not in the document]
Comment (Da-Colon) [reply src/providers/Fractal/governance/types.ts]: I've added a note comment

**Comment by Da-Colon** [comment]
switch to draft until I can circle back and update the `useTranslation` hook with libraries

**Review by mobiusTripper-crypto — APPROVED** [review]

**Pull request merged**